cmd/helmvm/token.go

```diff
@@ -83,14 +95,22 @@ var tokenCreateCommand = &cli.Command{
 		}
 		if !defaults.DecentralizedInstall() {
 			if err := defaults.SetInstallAsDecentralized(); err != nil {
-				return fmt.Errorf("failed to set decentralized install: %w", err)
+				return fmt.Errorf("unable to set decentralized install: %w", err)
 			}
 		}
+		token := JoinToken{ClusterID: metrics.ClusterID(), Token: buf.String(), Role: role}
+		encodedToken := bytes.NewBuffer(nil)
+		encoder := base64.NewEncoder(base64.StdEncoding, encodedToken)
+		marshaler := json.NewEncoder(encoder)
+		if err := marshaler.Encode(token); err != nil {
```

Comment:
How about an Encode() function on the JoinToken struct?

Reply:
that makes sense. done.